sections/06_discussion.md

```diff
@@ -475,17 +475,12 @@ sharing and analysis [@doi:10.1038/ng.3830].
 The sharing of high-quality, labeled datasets will be especially valuable.  In
 addition, researchers who invest time to preprocess datasets to be suitable for
 deep learning can make the preprocessing code (e.g. Basset
-[@tag:Kelley2016_basset] and variationanalysis [@tag:Torracinta2016_deep_snp])
```

Comment:
This actually should be `variationanalysis`, one word.  That's the official name of the tool, all in lowercase.